spec/requests/v1/pacbio/wells_spec.rb

```diff
@@ -127,7 +129,42 @@
           expect(Messages).to receive(:publish)
           post v1_pacbio_wells_path, params: body, headers: json_api_headers
         end
+      end
+
+      context 'on ccs mode with pre-extension success' do
+        let(:body) do
+          {
+            data: {
+              type: "wells",
+              attributes: {
+                wells: [
+                  { row: 'A',
+                    column: '1',
+                    movie_time: 8,
+                    insert_size: 8000,
+                    on_plate_loading_concentration: 8.36,
+                    sequencing_mode: 'CCS',
+                    pre_extension_time: 2,
+                    relationships: {
+                      plate: {
+                        data: {
+                          type: 'plate',
+                          id: plate.id
+                        }
+                      }
+                    }
+                  }
+                ],
+              }
+            }
+          }.to_json
+        end
 
+        it 'creates a well with pre-extension time set' do
```

Comment:
Could be worth testing with more than one well in the `wells` body, to make sure different wells with different CCS is inserted correctly

Reply:
Sure can do. What do you mean `with different CCS` - do you mean with different sequencing mode, or different pre-extension time? Thanks